fossology/fossology
# fix(packaging): Create apache softlink on source

### Changes

1. Move the softlink creation from Makefile to `fo-postinstall`.
2. Create softlink only when built from source.

## How to test

1. On source install:
    1. Check if running `fo-postinstall` enables fossology website on apache. (See #182)
1. On debian packaging:
    1. Check if packages can be built (`dpkg-buildpackage -I`).
    1. Check if installation of packages does not fail and fossology website is enabled in apache.
1. On docker:
    1. Check if docker image can be built.
    1. Check if a docker container can be started from the built image.

Closes #1661
